--- libhubbub-0.3.5/test/tokeniser2.c.orig	2018-09-24 18:14:23.674302100 -0400
+++ libhubbub-0.3.5/test/tokeniser2.c	2018-09-24 18:15:06.977567400 -0400
@@ -3,6 +3,7 @@
 #include <string.h>
 
 #include <json.h>
+#include <bits.h>
 
 #include <parserutils/input/inputstream.h>
 
--- libhubbub-0.3.5/test/tokeniser3.c.orig	2018-09-24 19:18:42.736591200 -0400
+++ libhubbub-0.3.5/test/tokeniser3.c	2018-09-24 19:23:23.310127200 -0400
@@ -3,6 +3,7 @@
 #include <string.h>
 
 #include <json.h>
+#include <bits.h>
 
 #include <parserutils/input/inputstream.h>
 
